bazel_tools/client_server_test/client_server_test.bzl

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@bazel_skylib//lib:shell.bzl", "shell")
```

Comment:
Is this still used? `shell.quote` seems to be inlined in `_expand_args`.

Reply:
thx, removed.